libs/ur-registry-ffi/src/extend/crypto_hd_key.rs

```diff
@@ -0,0 +1,140 @@
+use ur_registry::crypto_hd_key::CryptoHDKey;
+use ur_registry::crypto_account::CryptoAccount;
+use ur_registry::crypto_output::CryptoOutput;
+use ur_registry::traits::From;
+use anyhow::Error;
+use anyhow::format_err;
+use hex;
+use serde_json::json;
+use crate::export;
+use serde::{Serialize, Deserialize};
+use secp256k1::{Parity, XOnlyPublicKey};
+
+#[derive(Default, Clone, Debug, Serialize, Deserialize)]
+pub struct HDKey {
+    key: String,
+    chain_code: String,
+    source_fingerprint: String,
+    note: Option<String>,
+}
+
+#[derive(Default, Clone, Debug, Serialize, Deserialize)]
+pub struct MultiHDKeys {
+    hd_keys: Vec<HDKey>,
+}
+
+impl Into<HDKey> for CryptoHDKey {
+    fn into(self) -> HDKey {
+        HDKey {
+            key: hex::encode(self.get_key()),
+            chain_code: hex::encode(self.get_chain_code().unwrap_or_default()),
+            source_fingerprint: hex::encode(self.get_origin().unwrap_or_default().get_source_fingerprint().unwrap_or_default()),
+            note: self.get_note(),
+        }
+    }
+}
+
+impl Into<MultiHDKeys> for CryptoAccount {
+    fn into(self) -> MultiHDKeys {
+        let hd_keys = self.get_output_descriptors().iter()
+            .map(|output| CryptoOutput::get_hd_key(output).unwrap_or_default())
+            .map(|crypto_hd_key| crypto_hd_key.into())
+            .collect();
+        MultiHDKeys {
+            hd_keys: hd_keys,
+        }
+    }
+}
+
+export! {
+    @Java_com_keystone_sdk_KeystoneNativeSDK_getSourceHDKey
+    fn get_source_hd_key(
```

Comment:
What does `source` mean here?
Could we rename `get` to `parse` to follow the naming convention?